xhyve/xhyve.go

```diff
@@ -85,7 +85,7 @@ var (
 	ErrMachineNotExist = errors.New("machine does not exist")
 	diskRegexp         = regexp.MustCompile("^/dev/disk([0-9]+)")
 	kernelRegexp       = regexp.MustCompile(`(vmlinu[xz]|bzImage)[\d]*`)
-	kernelOptionRegexp = regexp.MustCompile(`(\t|\s{2})append`)
+	kernelOptionRegexp = regexp.MustCompile(`[\t|\s{2}]append\s+([[:print:]]+)`)
```

Comment:
I am not sure why not use `(^\t|\s{2})append\s+([[:print:]]+)` because that regex will avoiding matching `append something.` in https://regex101.com/r/LghmGX/4 and regex which you used will have match to that string.